### PR TITLE
[AMS-2025] Add new organizers

### PR DESCRIPTION
**Pull request by FloorD**
Adding several new organizers for Amsterdam 2025

*If you are adding or removing organizers*, send an email to info@devopsdays.org with the full names, email addresses, and organizations the added/removed persons represent. (The global team uses this information to maintain your mailing list as well as the Slack.)
I will do this shortly, but can also edit the ML myself.

### CODE REVIEW

**Review by issyl0 — COMMENTED** [review]
Yes, ordinarily I'd approve, but since I'm one of them I won't approve directly. Also it requires an email to Core to add to the mailing lists, which I'm not privy to.

File: static/events/2025-amsterdam/organizers/bram.png
Comment (toshywoshy) [inline static/events/2025-amsterdam/organizers/bram.png:1]: Please move to `assets/events/2025-amsterdam/organizers/`

File: static/events/2025-amsterdam/organizers/issy.jpeg
Comment (toshywoshy) [inline static/events/2025-amsterdam/organizers/issy.jpeg:1]: Please move to `assets/events/2025-amsterdam/organizers/`

File: static/events/2025-amsterdam/organizers/joep.png
Comment (toshywoshy) [inline static/events/2025-amsterdam/organizers/joep.png:1]: Please move to `assets/events/2025-amsterdam/organizers/`

File: static/events/2025-amsterdam/organizers/nati.jpg
Comment (toshywoshy) [inline static/events/2025-amsterdam/organizers/nati.jpg:1]: Please move to `assets/events/2025-amsterdam/organizers/`

File: static/events/2025-amsterdam/organizers/stephan.png
Comment (toshywoshy) [inline static/events/2025-amsterdam/organizers/stephan.png:1]: Please move to `assets/events/2025-amsterdam/organizers/`

**Review by toshywoshy — CHANGES_REQUESTED** [review]
Please move the images to the assets folder
And you need to mail info@

**Comment by FloorD** [comment]
> Please move the images to the assets folder And you need to mail info@

I already sent that email, and updated the group. Will move the pictures next.

**Comment by jerdog** [comment]
@FloorD I went ahead and merged the changes from main into the branch to kick-off the final checks

**Review by toshywoshy — APPROVED** [review]

**Pull request merged**